kd/deployment.yaml

```
@@ -160,10 +165,10 @@ spec:
                 name: hocs-queue-config
           env:
             - name: JAVA_OPTS
-              value: >-
-              -Xms768m -Xmx4096m -XX:+UseG1GC -Djavax.net.ssl.trustStore=/etc/keystore/truststore.jks
-              -Dhttps.proxyHost=hocs-outbound-proxy.{{.KUBE_NAMESPACE}}.svc.cluster.local
-              -Dhttps.proxyPort=31290 -Dhttp.nonProxyHosts=*.{{.KUBE_NAMESPACE}}.svc.cluster.local
+              value: >
+                -Xms768m -Xmx4096m -XX:+UseG1GC -Djavax.net.ssl.trustStore=/etc/keystore/truststore.jks
+                -Dhttps.proxyHost=hocs-outbound-proxy.{{.KUBE_NAMESPACE}}.svc.cluster.local
+                -Dhttps.proxyPort=31290 -Dhttp.nonProxyHosts=*.{{.KUBE_NAMESPACE}}.svc.cluster.local
```

Splitting non-semantically related configs into different lines:
```suggestion
                -Dhttps.proxyPort=31290 
                -Dhttp.nonProxyHosts=*.{{.KUBE_NAMESPACE}}.svc.cluster.local
```
(huh. do we not need to specify a HTTP proxy port as well as a HTTPS one?)